testsuite/backend/mockserver.py

```diff
@@ -35,9 +35,7 @@ def commit(self):
         )
         self.service.commit()
 
-    def wait_for_ready(self, timeout=300):
-        """Waits until Deployment is marked as ready"""
-        success = self.service.wait_until(
-            lambda obj: "ip" in self.service.refresh().model.status.loadBalancer.ingress[0], timelimit=timeout
-        )
-        assert success, f"Service {self.name} did not get ready in time"
+    def wait_for_ready(self, timeout=60 * 5):
```

Comment:
where do you actually call this method? I can see that you removed the `self.deployment.wait_for_ready()` call (was on line 26) and move it into this method but I am struggling to see where this method is actually used.

Reply:
The `MockserverBackend` is used only this this one tests and the `wait_for_ready()` is called here: https://github.com/azgabur/testsuite/blob/refactor-mockserver/testsuite/tests/singlecluster/gateway/dnspolicy/health_check/test_additional_headers.py#L48

Reply:
I must have been blinded by that bloody sun shining directly at my monitor :) All clear now, thanks!